docs/conf.py

```diff
@@ -1,78 +1,71 @@
 sys.path.insert(0, os.path.abspath("../"))
 
-from screenpy_playwright.__version__ import __version__  # noqa: need the path first
+from screenpy_playwright.__version__ import __version__, __author__, __copyright__
 
-autodoc_mock_imports = ["playwright", "screenpy", ]
+autodoc_mock_imports = ["playwright", "screenpy"]
 
 
 # -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'screenpy_playwright'
-copyright = '2022-2023, Perry Goy'
-author = 'Perry Goy'
-
-# The full version, including alpha/beta/rc tags
+project = "screenpy_playwright"
+copyright = __copyright__
+author = __author__
 release = __version__
 
 
 # -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
 extensions = [
-    "sphinx.ext.intersphinx",
     "sphinx.ext.autodoc",
     "sphinx.ext.autosectionlabel",
+    "sphinx.ext.intersphinx",
     "sphinx_rtd_theme",
     "autodoc_skip_protocols",
 ]
 
-intersphinx_mapping = {
-    "screenpy": ("https://screenpy-docs.readthedocs.io/en/latest/", None),
-    "selenium": ("https://selenium-python.readthedocs.io/", None),
-}
+templates_path = ['_templates']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+
+
+
+
 
-autodoc_member_order = "bysource"
 
-# Add any paths that contain templates here, relative to this directory.
-templates_path = ["_templates"]
 
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
 
 # -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
 html_theme = "sphinx_rtd_theme"
+html_static_path = []
 
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
-
-# Other HTML settings
 autodoc_member_order = "bysource"
 add_module_names = False
+
+
+# -- Options for intersphinx extension ---------------------------------------
+
+# Example configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {
+    "screenpy": ("https://screenpy-docs.readthedocs.io/en/latest/", None),
+    "selenium": ("https://selenium-python.readthedocs.io/", None),
```

Comment:
I don't think we need the Selenium docs in the Playwright one, sorry—this was my copy/paste mistake.

Reply:
Fixed